### DISCUSSION TITLE
Kurtarma ekipleri ve kordinasyon saglayicilari icin bir staff web paneli.

### DISCUSSION THREAD

**Discussion by umit-ozturk** [acikkaynak/deprem-yardim-projesi#55] [Ideas]
Kurtarma ekipleri ve kordinasyon saglayicilari icin bir staff web paneli olusturulmalidir. Afet aninda ve sonrasinda veri girisi yapabilecek personeller icin. Belirli dashboardlar metrikler ve veri girisi alanlari olabilir.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolü gerçekleştirildi.
Notları: "legal zemini gerekli onaylar alındıktan sonra oluşturulabilir."

**Comment by agitrubard** [comment]
@umit-ozturk Selamlar,
Bu konuları da içeren bir proje için discussion oluşturmuştuk. İsterseniz oraya dahil edebiliriz, bu sayede fikirler birbirini ezmemiş olur ve fikir birliği ile birlikte ortaya çok iyi şeyler çıkabilir. 

[https://github.com/acikkaynak/afet-org/discussions/35](https://github.com/acikkaynak/afet-org/discussions/35)

Bu konu üzerine hem fikir olursak Discord üzerinden bana ulaşabilirsiniz;
`agitrubard#7048`
